addons/web/static/src/views/fields/x2many/x2many_field.xml

```diff
@@ -5,12 +5,27 @@
         <div t-attf-class="o_field_x2many o_field_x2many_{{viewMode}}">
             <div class="o_x2m_control_panel">
                 <t t-if="displayAddButton">
```

Comment:
maybe rename this (it's not necessarily an Add button, and it could be more), for instance "displayControlPanelButtons".